tooling/nargo_fmt/src/formatter/attribute.rs

```diff
@@ -165,6 +172,35 @@ impl Formatter<'_> {
         self.write_right_bracket();
     }
 
+    fn format_cfg_attribute(&mut self, cfg_attribute: CfgAttribute) {
+        self.write_current_token_and_bump(); // #[
+        self.skip_comments_and_whitespace();
+        self.format_path(Path::from_single("cfg".to_string(), cfg_attribute.location()));
+        self.skip_comments_and_whitespace();
+        if self.is_at(Token::LeftParen) {
```

Comment:
Is there a case where a left paren wouldn't be at this position? (I think otherwise the parser will error and with errors the formatter doesn't run)